.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
```

Comment:
what is the difference between the title and the name, it seems like the same

Reply:
@Hmerin  only one concern about this, everything else is good to me.

Reply:
The first yaml part is used by GitHub to config the issue and it wont be shown to users, try to create an bug issue and you will notice it has some tags and name predefined. This part is to configure those settings.